ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js

```diff
@@ -0,0 +1,358 @@
+                                colorType="white"
+                                radiusType="circle"
+                                backgroundColorType="purple"
+                                border="none"
+                                fontSize="12px"
+                                fontFamily="InterSemiBold"
+                                onClick={() => window.open('https://docs.memphis.dev/memphis/dashboard-ui/integrations/notifications/slack', '_blank')}
+                            />
+                        </div>
+                    </div>
+                    {githubConfiguration.integrateDesc}
+                    <Form name="form" form={creationForm} autoComplete="off" className="integration-form">
+                        <div className="api-details">
+                            <p className="title">API Details</p>
+                            <div className="api-key">
+                                <span className="connect-bth-gh">
+                                    <p>API Token</p>
+                                    {isValue ? (
+                                        <Button
+                                            width="100px"
+                                            height="22px"
+                                            placeholder={
+                                                <span>
+                                                    <img src={tickCircle} className="connected" alt="connected" />
+                                                    &nbsp;Connected
+                                                </span>
+                                            }
+                                            colorType="white"
+                                            radiusType="circle"
+                                            backgroundColorType="green"
+                                            fontSize="12px"
+                                            fontFamily="InterSemiBold"
+                                            // isLoading={loadingSubmit}
+                                            // disabled={isValue && !creationForm.isFieldsTouched()}
+                                            onClick={createIntegration}
+                                        />
+                                    ) : (
+                                        <Button
+                                            width="70px"
+                                            height="22px"
+                                            placeholder={'Connect'}
+                                            colorType="white"
+                                            radiusType="circle"
+                                            backgroundColorType="purple"
+                                            fontSize="12px"
+                                            fontFamily="InterSemiBold"
+                                            // isLoading={loadingSubmit}
```

Comment:
When pressing connect, you can activate the loader inside the button

Reply:
Done